portfolio/src/main/webapp/index.html

```diff
@@ -10,11 +10,17 @@
     <div id="content">
       <h1 class="title">Jonathan Ward's Portfolio</h1>
       <h2 class="center">This is Jonathan's portfolio.</h2>
-      <div id="content">
-          <button class="btn" onclick="greeting()">Hello!</button>
-          <div id="data-container">
+      <form action="/data" method="POST">
+          <p> Enter a comment </p>
+          <textarea name="text-input">Enter a comment here</textarea>
+          <br/><br/>
+          <input type="submit"/>
+      </form>
+      <!-- <div class="centerbtn "id="content">
+          <button class="btn" onclick="retrieveComments()">Comments</button>
+          <div id="comment-container">
           </div>
-      </div>
+      </div> -->
```

Comment:
Typically, you'll want to remove commented out code. You can always see what used to be here by examining the git history.